docs/userguide/04_bespoke-executor.md

```diff
@@ -0,0 +1,141 @@
+(executor_chapter)=
+# BespokeExecutor
+
+BespokeFit not only provides tools and schemas for specifying workflows, but
+also an execution environment for running them. This environment is called
+[BespokeExecutor]. BespokeExecutor combines a HTTP server, job queue,
+parallel scheduler, and workflow interpreter so that workflows produced by
+BespokeFit can be efficiently executed in parallel.
+
+BespokeExecutor runs in the background on a computer with plenty of CPU power.
+This might be a local workstation or a remote HPC cluster. It then provides a
+server to which jobs can be submitted. Submitted jobs are placed in a queue
+until resources become available, at which point the executor runs them.
+BespokeExecutor can break a workflow down into stages and run parts in
+parallel with parts from other workflows so that even serial steps can take
+advantage of the host platform's parallelism. This means that several workflows
```

Comment:
This sentence is kinda hard to read, though I can't quite suggest a better wording for it.

Reply:
Rephrased. Thanks!